druid/src/widget/container.rs

```diff
@@ -135,17 +135,25 @@ impl<T: Data> Widget<T> for Container<T> {
     }
 
     fn paint(&mut self, ctx: &mut PaintCtx, data: &T, env: &Env) {
-        if let Some(background) = self.background.as_mut() {
-            let panel = ctx.size().to_rounded_rect(self.corner_radius);
+        let border_width = match &self.border {
+            Some(border) => border.width.resolve(env),
+            None => 0.0,
+        };
 
+        let inset_panel = ctx
+            .size()
+            .to_rect()
+            .inset(-1.0 * border_width)
+            .to_rounded_rect(self.corner_radius);
+
+        if let Some(background) = self.background.as_mut() {
             ctx.with_save(|ctx| {
-                ctx.clip(panel);
+                ctx.clip(inset_panel);
```

Comment:
Painting the background with these insets means that the background isn't visible with dashed borders etc. Is this behavior we want?